SwiftFlow/CoreTypes/Store.swift

```diff
@@ -8,13 +8,22 @@
 
 import Foundation
 
+/** 
```

Comment:
Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)